sklearn/inspection/_plot/decision_boundary.py

```diff
@@ -248,6 +247,12 @@ def from_estimator(
             For multiclass problems, :term:`predict` is selected when
             `response_method="auto"`.
 
+        class_of_interest : int, float, bool or str, default=None
+            The class considered when plotting the decision. By default,
+            `estimators.classes_[1]` is considered as the positive class.
```

Comment:
Once we implement the "max" strategy for multiclass, it can be made the default, that is when `class_of_interest=None`, in which case the docstring and tests will need to be updated accordingly.